Prose/ProseLib/Sources/ProseCoreStub/MessageStore.swift

```diff
@@ -15,18 +15,30 @@ public struct Message {
     public let senderId: JID
     public let content: String
     public let timestamp: Date
+
+    public init(
+        senderId: JID,
+        content: String,
+        timestamp: Date
+    ) {
+        self.senderId = senderId
+        self.content = content
+        self.timestamp = timestamp
+    }
 }
 
 public struct MessageStore {
     public var messages: (_ chatId: ChatID) -> [Message]?
     public var unreadMessages: () -> OrderedDictionary<ChatID, [Message]>
+    public var sendMessage: (_ chatId: ChatID, _ message: Message) -> Void
```

Comment:
This isn't used anymore, should I revert it?

Reply:
Sure